Assets/C#/PlayerScripts/PlayerAbilities/Ability_PointSpawner.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Ability_PointSpawner : Ability_ObjectSpawner {
+	public float spawnRange;
+
+	public override void SpawnSpell(PlayerComponent.Buf data)
+	{
+		Ray ray = Camera.main.ScreenPointToRay(Input.mousePosition);
```

Comment:
Please make sure you rewrite this so we aren't using the mouse position, but the vectors stored in PlayerComponent.Buf data